shared/net/bind/types.go

```diff
@@ -2,45 +2,45 @@ package bind
 
 import "net"
 
-// ListenerConfig mimicks net.ListenConfig providing a configuration
+// Listener mimics net.ListenConfig providing a configuration
 // context for net.Listen() and net.ListenPacket() alternatives
-type ListenerConfig interface {
+type Listener interface {
 	Listen(network, addr string) (net.Listener, error)
 	ListenPacket(network, addr string) (net.PacketConn, error)
 }
 
-// AllListenerConfig is equivalent to ListenerConfig but takes an
+// AllListener is equivalent to ListenerConfig but takes an
 // array of addresses
-type AllListenerConfig interface {
+type AllListener interface {
 	ListenAll(network string, addr []string) ([]net.Listener, error)
 	ListenAllPacket(network string, addr []string) ([]net.PacketConn, error)
 }
 
-// TCPListenerConfig provides a context-aware alternative to net.ListenTCP
-type TCPListenerConfig interface {
+// TCPListener provides a context-aware alternative to net.ListenTCP
+type TCPListener interface {
 	ListenTCP(network string, laddr *net.TCPAddr) (*net.TCPListener, error)
 }
 
-// AllTCPListenerConfig is equivalent to TCPListenerConfig but takes an
+// AllTCPListener is equivalent to TCPListenerConfig but takes an
 // array of addresses
-type AllTCPListenerConfig interface {
+type AllTCPListener interface {
 	ListenAllTCP(network string, ladders []*net.TCPAddr) ([]*net.TCPListener, error)
 }
 
-// UDPListenerConfig provides a context-aware alternative to net.ListenUDP
-type UDPListenerConfig interface {
+// UDPListener provides a context-aware alternative to net.ListenUDP
+type UDPListener interface {
 	ListenUDP(network string, laddr *net.UDPAddr) (*net.UDPConn, error)
 }
 
-// AllUDPListenerConfig is equivalent to UDPListenerConfig but takes an
+// AllUDPListener is equivalent to UDPListenerConfig but takes an
```

Comment:
Se above, you need to correct the comment to be consistent

Reply:
amended